pkg/model/openstackmodel/servergroup.go

```diff
@@ -60,7 +60,7 @@ func (b *ServerGroupModelBuilder) buildInstances(c *fi.ModelBuilderContext, sg *
 	}
 	igMeta["k8s"] = b.ClusterName()
 	igMeta["KopsInstanceGroup"] = ig.Name
-	igMeta["KopsRole"] = fmt.Sprintf("%s", ig.Spec.Role)
+	igMeta["KopsRole"] = ig.Spec.Role.String()
```

Comment:
This is the line that's actually causing the error that travis is finding.  I think you can just do `string(ig.Spec.Role)`

Reply:
OK I will change it